### PR TITLE
Channel color int

### PR DESCRIPTION
**Pull request by will-moore**
Fixes https://github.com/openmicroscopy/omero-marshal/issues/35

We use the rgba -> integer helpers from #31 to set 'Color' integer on Channels.

### CODE REVIEW

**Comment by will-moore** [comment]
Travis is failing with 
```
$ omego download --ice 3.5 --release ${OMERO_VERSION} py
Traceback (most recent call last):
...
    "No artifacts found for ice version: %s" % ice_ver)
AttributeError: No artifacts found for ice version: ice35
```
Will try again...

**Comment by sbesson** [comment]
@will-moore: I am not sure what went wrong overall in the `omego/latest redirects` logic but this failure seems to be a direct consequence of the 5.3.0-m3 release as ` ~/venvs/omego/bin/omego download --ice 3.5 --release 5.3.0-m7 py -vv` works for me but ` ~/venvs/omego/bin/omego download --ice 3.5 --release 5.3.0-m8 py -vv` fails /cc @jburel

**Comment by sbesson** [comment]
@will-moore in the interim, can you change https://github.com/openmicroscopy/omero-marshal/blob/master/.travis.yml#L19 to use `5.3.0-m7` instead of `5.3` ? This should download the previous milestone artifacts fine with Ice 3.5 and restore Travis

**Comment by jburel** [comment]
Changing the default to ice 3.6 in travis to match the requirements will be more suitable

**Comment by jburel** [comment]
We will do the switch to ice 3.6 in another PR.
For now @will-moore you need to point to 5.3.0-m7 as suggested by @sbesson to fix the travis failure.
This is due to a limitation of omego see https://github.com/ome/omego/pull/101

**Comment by jburel** [comment]
I will also test with signed integer for the color

**Comment by will-moore** [comment]
@sbesson Yes, RGBA/int conversion is tested ```test_well_color_decoder```  see https://github.com/openmicroscopy/omero-marshal/pull/31/commits/3145eb2925e98d98ab85169c1a29a665dd76b55e

File: .travis.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ env:
   matrix:
     - OMERO_VERSION=5.1
     - OMERO_VERSION=5.2
-    - OMERO_VERSION=5.3
+    - OMERO_VERSION=5.3.0-m7
```
Comment (jburel) [inline .travis.yml]: This commit will have to be reverted when omego is sorted

**Pull request merged**